index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+</head>
+<body>
+  
+
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+    
+         <nav>
+           <ul>
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b><img src="images/men.PNG" alt="ss"></b></a>
+             </li>
+        
+           </ul>
+        </nav>
+    </header>
+
+    <section class="section1">
+          <img src="images/l1.png" alt="abc" class="l1">
+          <img src="images/m.png" alt="abc" class="m">
+          <img src="images/o.png" alt="abc" class="m kap">
+
+          <img src="images/r6.png"  alt="abc" class="r6" >
+          <img src="images/e3.png"  alt="abc" class="image3 dkl" >
+    </section>
```

Comment:
- [ ] Please note that all images meant for decorations should be added as background image in CSS as this is best practice. Check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to get more details. I suggest you use multiple background images and position each of them using the `background-position` property. You can get more info about this from [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds).